applications/NXreftof.nxdl.xml

```diff
@@ -27,6 +27,21 @@
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xsi:schemaLocation="http://definition.nexusformat.org/nxdl/3.1 ../nxdl.xsd"
   >
+  	<symbols>
+      <doc>
+	    The symbol(s) listed here will be used below to coordinate datasets
+		with the same shape.
+      </doc>
+      <symbol name="xSize">
+	    <doc>xSize description</doc>
```

Comment:
Is this the number of X values?  (`Size description` is vague)

Reply:
when a definition didn't specify a symbol table and I created one for it I took whatever doc string was associated with the symbol where it was used and put that in the doc string for the symbol in the symbols table, when there was no doc string to use I was reluctant to make one up based on my understanding of how the symbol was being used, so as a placeholder I was just putting symbol_name description and was going to let the contributor correct the doc string in the table